wp-content/themes/pub/wporg-learn-2020/functions.php

```diff
@@ -290,6 +290,42 @@ function wporg_archive_modify_query( WP_Query $query ) {
 }
 add_action( 'pre_get_posts', 'wporg_archive_modify_query' );
 
+/**
+ * Modify the workshop post type archive query to prioritize workshops in the user's locale.
+ *
+ * @param array    $clauses
+ * @param WP_Query $query
+ *
+ * @return array
+ */
+function wporg_archive_query_prioritize_locale( $clauses, $query ) {
+	if ( ! $query->is_post_type_archive( 'wporg_workshop' ) || is_admin() ) {
+		return $clauses;
+	}
+
+	global $wpdb;
+
+	$locale      = get_locale() ?: 'en_US';
```

Comment:
I don't think the fallback is necessary, except maybe if we're worried about a `locale` callback making it falsy.

https://github.com/WordPress/WordPress/blob/8ef335a566cbf862263c4c6efd4e130ea976b477/wp-includes/l10n.php#L69-L71

Reply:
Ah you're right. Fixed in 518848d